data/bss-factory-sets.json

```diff
@@ -0,0 +1,5689 @@
+        "evs": [156, 0, 252, 0, 100, 0],
+        "teratype": ["water", "fire", "electric", "dark"],
+        "ability": "unaware"
+      }
+    ]
+  },
+  "meowscarada": {
+    "weight": 5,
+    "sets": [
+      {
+        "species": "meowscarada",
+        "weight": 25,
+        "moves": [
+          ["flowertrick"],
+          ["knockoff"],
+          ["uturn", "trick"],
+          ["playrough", "suckerpunch", "lowkick"]
+        ],
+        "item": ["choicescarf"],
+        "nature": "jolly",
+        "evs": [4, 252, 0, 0, 0, 252],
+        "teratype": ["grass", "dark"],
+        "ability": "protean"
+      },
+      {
+        "species": "meowscarada",
+        "weight": 10,
+        "moves": [
+          ["flowertrick"],
+          ["knockoff"],
+          ["uturn", "trick"],
+          ["terablast"]
+        ],
+        "item": ["choicescarf"],
+        "nature": "jolly",
+        "evs": [4, 252, 0, 0, 0, 252],
+        "teratype": ["rock", "ghost"],
+        "wantsTera": true,
+        "ability": "protean"
+      },
+      {
+        "species": "meowscarada",
+        "weight": 30,
+        "moves": [
+          ["flowertrick"],
+          ["knockoff"],
+          ["suckerpunch", "taunt"],
+          ["toxicspikes"]
+        ],
+        "item": ["focussash"],
+        "nature": "jolly",
+        "evs": [0, 252, 4, 0, 0, 252],
+        "teratype": ["grass"],
+        "ability": "overgrow"
+      },
+      {
+        "species": "meowscarada",
+        "weight": 20,
+        "moves": [
+          ["flowertrick"],
+          ["knockoff"],
+          ["terablast"],
+          ["suckerpunch", "toxicspikes"]
+        ],
+        "item": ["focussash"],
+        "nature": "jolly",
+        "evs": [0, 252, 4, 0, 0, 252],
+        "teratype": ["rock", "ghost"],
+        "wantsTera": true,
+        "ability": "overgrow"
+      },
+      {
+        "species": "meowscarada",
+        "weight": 10,
+        "moves": [
+          ["flowertrick"],
+          ["knockoff"],
+          ["uturn", "trick"],
+          ["suckerpunch"]
+        ],
+        "item": ["choiceband"],
+        "nature": "jolly",
+        "evs": [4, 252, 0, 0, 0, 252],
+        "teratype": ["grass"],
+        "ability": "protean"
+      },
+      {
+        "species": "meowscarada",
+        "weight": 5,
+        "moves": [
+          ["flowertrick"],
+          ["knockoff"],
+          ["uturn", "trick", "suckerpunch"],
+          ["terablast"]
+        ],
+        "item": ["choiceband"],
+        "nature": "jolly",
+        "evs": [4, 252, 0, 0, 0, 252],
+        "teratype": ["rock", "ghost"],
+        "wantsTera": true,
+        "ability": "protean"
+      }
+    ]
+  },
+  "azumarill": {
+    "weight": 5,
+    "sets": [
+      {
+        "species": "azumarill",
+        "weight": 45,
+        "moves": [["aquajet"], ["playrough"], ["liquidation"], ["superpower"]],
+        "item": ["assaultvest"],
+        "nature": "adamant",
+        "evs": [252, 252, 0, 0, 0, 4],
```

Comment:
Please don't put the 4 in Speed on max/max sets in randoms. Move it elsewhere. Also applies to other Azumarill sets. Move the 12s, too.